### DISCUSSION TITLE
Allow access to client if user not logged in

### DISCUSSION THREAD

**Discussion by moonwalk5** [Ylianst/MeshCentral#5771] [Q&A]
Hello Together,

we have set strictly consent for all client access in "config.json".
```
{
    "userConsentFlags": {
        "desktopnotify": true,
        "terminalnotify": true,
        "filenotify": true,
        "desktopprompt": true,
        "terminalprompt": true,
        "fileprompt": true,
        "desktopprivacybar": true
    }
}
```
This is necessary by workers council, that we do not control the employees silently.

How can I allow the client access in the case, that the user is logged off. In this case, no user would be interfere.

Thanks in advance.

With best regards,

Jörg

**Comment by si458** [comment]
you cant...
the is a feature request for this but it hasnt been implemented yet https://github.com/Ylianst/MeshCentral/issues/4154
the best you can do is set `consentTimeout: 30` and `autoAcceptOnTimeout: true` under `consentMessages` under your `domains` in `config.json`
BUT
this will autoaccept the connection after 30 seconds of the user not consenting, REGALRDLESS of if the user is logged in OR not

**Comment by moonwalk5** [comment]
Hello Simon,

thanks for this answer. In this case I have to wait, otherwise I get a problem with the workers council and the GDPR rules.

With best regards,

Jörg

**Comment by ghost** [comment]
Hello,

i am issueing the same problem. Adminstring a handful of headless servers i need access without user consent. I tried the settings as opposed above but it is not working. The Connection is not accepted.

I tried:

      "consentMessages": {
        "title": "MeshCentral",
        "desktop": "{0} fragt: Soll Remote-Zugriff auf den Desktop gewährt werden?",
        "terminal": "{0} fragt: Soll Remote-Zugriff auf das Terminal gewährt werden?",
        "files": "{0} fragt: Soll Remote-Zugriff auf Dateien gewährt werden?",
        "consentTimeout": 30,
        "autoAcceptOnTimeout": true
      },

Thanks in advance,

Andree

**Reply by si458** [reply]
In theory I think u can right click the connect button and say without consent? 
I will have to checkup on this
